source/Body.cpp

```diff
@@ -263,15 +264,15 @@ void Body::SetSwizzle(int swizzle)
 // a sprite instead of a full animation data structure.
 void Body::SetFrameRate(float framesPerSecond)
 {
-	frameRate = framesPerSecond / 60.f;
+	frameRate = framesPerSecond / FRAME_RATE;
```

Comment:
Note that here and several other places, you are changing a float to a int which means there is now int->float conversion whenever these lines gets executed. In most cases this will basically not be noticeable.